.circleci/config.yml

```diff
@@ -7,24 +7,24 @@ jobs:
     steps:
       - checkout
       - run:
-          command: npm ci
+          command: yarn
```

Comment:
I think this should be `yarn install --frozen-lockfile` to prevent yarn from auto updated anything in `yarn.lock` when running ci testing.